release/models/devices-manifest/openconfig-terminal-device-property-types.yang

```diff
@@ -0,0 +1,459 @@
+      identity";
+  }
+
+  identity MODULATION_FORMAT_DPSK {
+    base MODULATION_FORMAT;
+    description
+      "Differential phase-shift keying (DPSK) modulation format
+      identity";
+  }
+
+  identity MODULATION_FORMAT_QPSK {
+    base MODULATION_FORMAT;
+    description
+      "Quadrature phase-shift keying (QPSK) modulation format
+      identity";
+  }
+
+  identity MODULATION_FORMAT_DP_QPSK {
+    base MODULATION_FORMAT;
+    description
+      "Dual-Polarization Quadrature Phase-Shift Keying (QPSK)
+      modulation format identity";
+  }
+
+  identity MODULATION_FORMAT_8QAM {
+    base MODULATION_FORMAT;
+    description
+      "8-Quadrature Amplitude Modulation (QAM) identity";
+  }
+
+  identity MODULATION_FORMAT_DP_8QAM {
+    base MODULATION_FORMAT;
+    description
+      "Dual-Polarization 8-Quadrature Amplitude Modulation (QAM)
+      identity";
+  }
+
+  identity MODULATION_FORMAT_DC_DP_8QAM {
+    base MODULATION_FORMAT;
+    description
+      "Dual-Carrier Dual-Polarization 8-Quadrature Amplitude
+      Modulation identity";
+  }
+
+  identity MODULATION_FORMAT_16QAM {
+    base MODULATION_FORMAT;
+    description
+      "16-Quadrature Amplitude Modulation (QAM) identity";
+  }
+
+  identity MODULATION_FORMAT_DP_16QAM {
+    base MODULATION_FORMAT;
+    description
+      "Dual-Polarization 16-Quadrature Amplitude Modulation (QAM)
+      identity";
+  }
+
+  identity MODULATION_FORMAT_DC_DP_16QAM {
+    base MODULATION_FORMAT;
+    description
+      "Dual-Carrier Dual-Polarization 16-Quadrature Amplitude Modulation
+      (QAM) identity";
+  }
+
+  identity MODULATION_FORMAT_32QAM {
+    base MODULATION_FORMAT;
+    description
+      "32-Quadrature Amplitude Modulation (QAM) identity";
+  }
+
+  identity MODULATION_FORMAT_DP_32QAM {
+    base MODULATION_FORMAT;
+    description
+      "Dual-Polarization 32-Quadrature Amplitude Modulation (QAM)
+      identity";
+  }
+
+  identity MODULATION_FORMAT_DC_DP_32QAM {
+    base MODULATION_FORMAT;
+    description
+      "Dual-Carrier Dual-Polarization 32-Quadrature Amplitude Modulation
+      (QAM) identity";
+  }
+
+  identity MODULATION_FORMAT_64QAM {
+    base MODULATION_FORMAT;
+    description
+      "64-Quadrature Amplitude Modulation (QAM) identity";
+  }
+
+  identity MODULATION_FORMAT_DP_64QAM {
+    base MODULATION_FORMAT;
+    description
+      "Dual-Polarization 64-Quadrature Amplitude Modulation (QAM)
+      identity";
+  }
+
+  identity MODULATION_FORMAT_DC_DP_64QAM {
+    base MODULATION_FORMAT;
+    description
+      "Dual-Carrier Dual-Polarization 64-Quadrature Amplitude Modulation
+      (QAM) identity";
+  }
+
+  identity MODULATION_FORMAT_PAM4 {
+    base MODULATION_FORMAT;
+    description
+      "4-level Pulse Amplitud Modulation (PAM)";
+  }
+
+  identity MODULATION_FORMAT_PAM8 {
+    base MODULATION_FORMAT;
+    description
+      "8-level Pulse Amplitud Modulation (PAM)";
+  }
+
+  identity MODULATION_FORMAT_PCS {
```

Comment:
I don't believe PCS is a modulation format?

Reply:
Removed in the latest commit.